products/developments/housing_by_cb/metadata.yml

```diff
@@ -0,0 +1,219 @@
+id: housing_database_by_2020_census_block
+
+attributes:
+  description: |-
+    Net change in housing units arising from new buildings, demolitions, or alterations for NYC Census Blocks since 2010.  The NYC Department of City Planning's (DCP) <a href="https://www1.nyc.gov/site/planning/data-maps/open-data/dwn-housing-database.page#housingdevelopmentproject">Housing Database</a> provide the 2010 census count of housing units, the net change in Class A housing units since the census, and the count of units pending completion for commonly used political and statistical boundaries. These tables are aggregated from the DCP Housing Database, which is derived from Department of Buildings (DOB)-approved housing construction and demolition jobs filed or completed in NYC since January 1, 2010. Net housing unit change is calculated as the sum of all three construction job types that add or remove residential units: new buildings, major alterations, and demolitions, and can be used to determine the change in legal housing units across time and space.
+    All previously released versions of this data are available at a <a href="https://www1.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts[year]=0"BYTES of the BIG APPLE- Archive</a>
```

Comment:
snippets!

Reply:
Also, I can never quite remember the syntax for yaml multi-line strings, so I reference this legendary [SO post](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) quite often. TLDR, the convention I've been shooting for is (on the socrata page) having a blank line between the description and the bytes link.